src/pyubx2/ubxreader.py

```diff
@@ -117,15 +119,16 @@ def read(self) -> tuple:
         'protfilter' determines which protocols are parsed.
         'quitonerror' determines whether to raise, log or ignore parsing errors.
 
+        :param bool parsing: a boolean flag to determine if the stream needs to be parsed

```

Comment:
can remove this docstring entry as it no longer applies

Reply:
Removed